libff/algebra/curves/alt_bn128/alt_bn128_pairing.cpp

```diff
@@ -238,56 +242,58 @@ alt_bn128_GT alt_bn128_final_exponentiation(const alt_bn128_Fq12 &elt)
 
 /* ate pairing */
 
-void doubling_step_for_flipped_miller_loop(const alt_bn128_Fq two_inv,
-                                           alt_bn128_G2 &current,
-                                           alt_bn128_ate_ell_coeffs &c)
+void doubling_step_for_flipped_miller_loop(
+    const alt_bn128_Fq two_inv,
+    alt_bn128_G2 &current,
+    alt_bn128_ate_ell_coeffs &c)
 {
     const alt_bn128_Fq2 X = current.X, Y = current.Y, Z = current.Z;
 
-    const alt_bn128_Fq2 A = two_inv * (X * Y);                     // A = X1 * Y1 / 2
-    const alt_bn128_Fq2 B = Y.squared();                           // B = Y1^2
-    const alt_bn128_Fq2 C = Z.squared();                           // C = Z1^2
-    const alt_bn128_Fq2 D = C+C+C;                                 // D = 3 * C
-    const alt_bn128_Fq2 E = alt_bn128_twist_coeff_b * D;             // E = twist_b * D
-    const alt_bn128_Fq2 F = E+E+E;                                 // F = 3 * E
-    const alt_bn128_Fq2 G = two_inv * (B+F);                       // G = (B+F)/2
-    const alt_bn128_Fq2 H = (Y+Z).squared() - (B+C);               // H = (Y1+Z1)^2-(B+C)
-    const alt_bn128_Fq2 I = E-B;                                   // I = E-B
-    const alt_bn128_Fq2 J = X.squared();                           // J = X1^2
-    const alt_bn128_Fq2 E_squared = E.squared();                   // E_squared = E^2
-
-    current.X = A * (B-F);                                       // X3 = A * (B-F)
-    current.Y = G.squared() - (E_squared+E_squared+E_squared);   // Y3 = G^2 - 3*E^2
-    current.Z = B * H;                                           // Z3 = B * H
-    c.ell_0 = alt_bn128_twist * I;                                 // ell_0 = xi * I
-    c.ell_VW = -H;                                               // ell_VW = - H (later: * yP)
-    c.ell_VV = J+J+J;                                            // ell_VV = 3*J (later: * xP)
+    const alt_bn128_Fq2 A = two_inv * (X * Y);           // A = X1 * Y1 / 2
+    const alt_bn128_Fq2 B = Y.squared();                 // B = Y1^2
+    const alt_bn128_Fq2 C = Z.squared();                 // C = Z1^2
+    const alt_bn128_Fq2 D = C + C + C;                   // D = 3 * C
+    const alt_bn128_Fq2 E = alt_bn128_twist_coeff_b * D; // E = twist_b * D
+    const alt_bn128_Fq2 F = E + E + E;                   // F = 3 * E
+    const alt_bn128_Fq2 G = two_inv * (B + F);           // G = (B+F)/2
+    const alt_bn128_Fq2 H = (Y + Z).squared() - (B + C); // H = (Y1+Z1)^2-(B+C)
+    const alt_bn128_Fq2 I = E - B;                       // I = E-B
+    const alt_bn128_Fq2 J = X.squared();                 // J = X1^2
+    const alt_bn128_Fq2 E_squared = E.squared();         // E_squared = E^2
+
+    current.X = A * (B - F); // X3 = A * (B-F)
+    current.Y =
+        G.squared() - (E_squared + E_squared + E_squared); // Y3 = G^2 - 3*E^2
```

Comment:
These comments at the end of the line force breaks where there should be. I wonder if we can run a script to mov these above the line...
Oh I actually just Googled it and saw this link: https://stackoverflow.com/questions/33878086/how-to-automatically-move-trailing-comments-to-the-line-above

I'll try some vim commands to see if this can make our life easier...

Reply:
Moved these all to the previous line.